SPECS/systemd/systemd.spec

```diff
@@ -856,6 +857,8 @@ install -Dm0644 10-timeout-abort.conf.user %{buildroot}%{user_unit_dir}/service.
 # https://fedoraproject.org/wiki/Changes/IncreaseVmMaxMapCount
 install -Dm0644 -t %{buildroot}%{_prefix}/lib/sysctl.d/ %{SOURCE17}
 
+install -Dm0644 -t %{buildroot}%{_prefix}/lib/sysctl.d/ %{SOURCE18}
```

Comment:
Should an entry also be added to the %files section? And a similar entry in %clean?

Reply:
> Should an entry also be added to the %files section?

not needed, `split-files.py` puts all sysctl files into `systemd-udev`

> And a similar entry in %clean?

not needed